MAINTAINING.md

```diff
@@ -155,6 +155,28 @@ GitHub's review sidebar.
 If there are multiple maintainers reviewing a PR, always give the other
 maintainers reasonable time to ACK before dismissing their review.
 
+### Release, Feature Freeze, and Backports
+
+Before the official release of a new RIOT version, two feature freeze periods
+are announced on the
+[RIOT development email list](https://lists.riot-os.org/mailman/listinfo/devel):
+The soft feature freeze and the hard feature freeze. During the soft feature
+freeze only PRs with minor impact should be merged into master. The hard feature
+freeze begins when the release manager creates a new release branch. Therefore,
+the restriction on merging PRs into the master branch are lifted at that point.
+
+Once the release branch is created, no new features should be backported.
+Instead, backports should consist only of bug fixes or of reverting features
+that were added during the last development period (and not part of any
+release), but didn't reach the required maturity or API stability yet. For
+bigger changes (which explicitly includes any revert), the PR has to be
+announced to the maintainer mailing list and should be merged no sooner than
+48h after the announcement and needs at least to ACKs.
```

Comment:
Here the formulation to me means that the contributor opening the PR containing a backported bugfix of high impact is not required to post it to the maintainer mailing list.(And also no maintainer reviewing it is explicitly required to do so. It is only required that no one hits the merge button before the PR has been announced and the 48h have passed.

To me this makes a lot of sense: Again, the contributor should not be required to know the release process. Also, a maintainer might to a partial review - maybe even only adding label to categorizes - and does not realize that the PR could be considered as having a high impact.

But I think it is fair to expect maintainers to not hit the merge button on PRs targeting a release branch if the cannot reasonably decide themselves if a PR is a high impact PR or not.

Reply:
I think if a new feature turned out to cause a regression in existing code, it should be reverted without much discussion.
The feature is then postponed to the next release if the bug has been fixed, but in a hard freeze you don't want to have 'scary' things.

Reply:
> I think if a new feature turned out to cause a regression in existing code, it should be reverted without much discussion.

For me this depends if the regression is hard/easy to fix, it would depend on contributors reactivity. But ultimately this is a Release manager decision, discussion can happen but if the manger considers it 'scary', the I agree there should not be much discussion.